docs/models.rst

```diff
@@ -87,7 +96,8 @@ We currently support single (non-composite) primary keys of any indexable field
 
 One must define a primary key by setting a ``pk`` parameter to ``True``.
 
-If you don't define a primary key, we will create a primary key of type ``IntField`` with name of ``id`` for you.
+If you don't define a primary key, we will create a primary key of type
+``IntField`` with name of ``id`` for you.
```

Comment:
I'm not sure shortening these has any benefit. ReST ignores insignificant whitespace (Newlines are only significant when there is at least 2) and we have a project-wide "standard" of about-100-characters wide.

Reply:
Hi @grigi, ok. I overread the 100-character-wide standard. I used doc8 to test the quality of the rst file and than you get 'line-too-long' errors. But I can live with a 100-character-wide rule. Do you want we to revert the line breaks? Or keep them here and I will never touch a long line again in the future?

Reply:
Please revert the line breaks, it makes it really difficult to find if there is any changes.
Of course fixing up a few ridiculous ones is fine :-)